src/utils/utils.js

```diff
@@ -11,18 +11,126 @@ const disabilityMap = {
   AUTYZM: "z autyzmem, w tym z zespołem Aspergera"
 };
 
-const calculateValues = values => {
-  const reason = values.applicant.reason;
-  let secondReason = values.applicant.secondReason;
-  if (reason && secondReason) {
-    const firstReason = disabilityMap[reason];
-    secondReason = disabilityMap[secondReason];
-    values.applicant.multipleDisability = [firstReason, secondReason];
-    values.applicant.reason = "SPRZEZONA";
-  }
-  return values;
-};
+class DocumentData {
+  constructor(values) {
+    this.values = values;
+  }
+
+  isMultipleDisability() {
+    return !!(
+      this.values.applicant.reason && this.values.applicant.secondReason
+    );
+  }
+
+  reason() {
+    if (this.isMultipleDisability()) {
+      return "SPRZEZONA";
+    }
+    return this.values.applicant.reason;
+  }
+
+  multipleDisability() {
+    if (!this.isMultipleDisability) {
+      return [];
+    }
+    const firstReasonDescription = disabilityMap[this.values.applicant.reason];
+    const secondReasonDescription =
+      disabilityMap[this.values.applicant.secondReason];
+    return [firstReasonDescription, secondReasonDescription];
+  }
+
+  applicantsList() {
+    const applicants = [
+      {
+        firstName: this.values.applicant.firstName1,
+        lastName: this.values.applicant.lastName1,
+        address: this.values.applicant.address1,
+        postalCode: this.values.applicant.postalCode1
```

Comment:
Maybe to avoid a lot of references to `this value`, at the top of function you could add
```javascript
const { applicantData } = this.values.applicant
```
and then
```javascript
firstName: applicant.firstName1,
```
It should be short and more comfortable.